timbermill-java/timbermill-local/src/main/java/com/datorama/oss/timbermill/common/ElasticsearchUtil.java

```diff
@@ -25,7 +25,7 @@
 					  "if (params.orphan != null && !params.orphan) {"
 					+ "    ctx._source.orphan = false;"
 					+ "}        "
-					+ "if (params.dateToDelete != null) {"
+					+ "if (params.dateToDelete != null && !ctx._source.status.equals( \\\"SUCCESS\\\") && !ctx._source.status.equals( \\\"UNTERMINATED\\\")) {"
```

Comment:
Need to add ERROR status as well as this is also a status that got a start event
Also please add a test for that scenario